### DISCUSSION TITLE
Issue with Launching Chrome with Specific User Profile in Selenium Base

### DISCUSSION THREAD

**Discussion by joako233** [seleniumbase/SeleniumBase#2825] [Q&A] [answered]
Hi Michael,

I'm encountering an issue when trying to open Chrome with a specific user profile using Selenium Base. Despite following the usual procedures, I can't seem to get it to work. Could you provide guidance on how to correctly launch Chrome with a specified profile using Selenium Base? I'm not sure if it's an error in my code or a general mistake on my part.
I can get the browser to open, but it opens with a profile that is not the specified path, instead using a default profile generated by Chrome.

Thanks for your help!

`from seleniumbase import Driver
perfil = r"C:\\Users\\joaqu\\AppData\\Local\\Google\\Chrome\\User Data\\Profile 11"

driver = Driver(user_data_dir=perfil)

input()`
![image](https://github.com/seleniumbase/SeleniumBase/assets/141453948/466dad01-c820-4c3b-ae9b-90a6235d7bab)

**Comment by mdmintz** [comment, marked as answer]
Duplicate of https://github.com/seleniumbase/SeleniumBase/issues/2432#issuecomment-1892459152 / https://github.com/seleniumbase/SeleniumBase/discussions/2331#discussioncomment-7730581

If you need multiple profiles, set a different `user_data_dir` for each one that you want. Only the `Default` profile is supported.

If you're setting the `user_data_dir` while using UC Mode, then make sure you read https://github.com/seleniumbase/SeleniumBase/discussions/2118#discussioncomment-7029201 first. (Don't mix a regular `user_data_dir` with one that is used by UC Mode Chrome, as the two formats are not compatible.)

**Reply by joako233** [reply]
Sorry but that is not my case, please see the code that I paste on the discussion.

**Reply by mdmintz** [reply]
The `user_data_dir` is not the same as the Profile directory. You can set a `user_data_dir`, but it will use the `Default` profile in there.

**Reply by joako233** [reply]
Sorry if my question is absurd, I’m a beginner. I don’t understand what is meant by: “The user_data_dir is not the same as the Profile directory. You can set a user_data_dir, but it will use the Default profile in there.” Thanks!

**Reply by mdmintz** [reply]
The User Data Dir is one level above the Profile Dir. (Inside the User Data Dir is the Profile Dir.) In your example, `User Data` was the User Data Dir, and `Profile 11` was the Profile Dir. The default Profile Dir is named `Default`. You can set your User Data Dir, and the `Default` profile will be automatically used. If you need to use multiple profiles, then use a separate User Data Dir for each one.

**Reply by joako233** [reply]
![image](https://github.com/seleniumbase/SeleniumBase/assets/141453948/66955abd-ab1b-42b1-bd7b-4936a897ae95)
I'm still unable to open the specific Chrome profile I want. I've tried this and it doesn't work either.

**Reply by mdmintz** [reply]
As I mentioned earlier, you can't specify a Profile Directory, but you can choose a User Data Dir (and the `Default` profile will be used).
